Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -322,6 +347,8 @@ class EditorTestSuite:
     use_null_renderer = True
     # Maximum time for a single editor to stay open on a shared test
     timeout_editor_shared_test = 300
+    # Marks the test as a MaterialEditor test instead of an Editor test.
+    material_editor_test = False
```

Comment:
Commented above, this is not extensible.

Reply:
yep will be changing it to class inheritance from EditorTestSuite